src/Configuration/src/CloudFoundry/ForwardedHeadersSettings.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+namespace Steeltoe.Configuration.CloudFoundry;
+
+public sealed class ForwardedHeadersSettings
```

Comment:
The name *Settings is misleading, because Aspire uses that to indicate it's _not_ an `IOptions`.

If this type is going to stay, it should be added to the configuration schema.